### PR TITLE
Introduction episode

### PR DESCRIPTION
**Pull request by raar1**
This adds a first draft of the introduction episode (01).

I have tried to go through the idea of a computational model, and why you would couple them in the first place, but I expect everyone here will have strong opinions on my reasoning. Please pay close attention to the exercise answers too, as I think I could use a bit of help with clarity there.

There is also a picture of CFD on an A380.

### CODE REVIEW

File: episodes/01-introduction.md [hunk 1]
```diff
@@ -1,101 +1,94 @@
...
-This is a lesson created via The Carpentries Workbench. It is written in
-[Pandoc-flavored Markdown](https://pandoc.org/MANUAL.txt) for static files and
-[R Markdown][r-markdown] for dynamic files that can render code into output. 
-Please refer to the [Introduction to The Carpentries 
-Workbench](https://carpentries.github.io/sandpaper-docs/) for full documentation.
+## Mathematical model vs computational model
+For this course, it is important to distinguish between a _mathematical_ model, and a _computational_ model.
+A mathematical model refers to the set of mathematical equations that describe the evolution of a given physical system.
+The computational model refers to the actual computer program that describes the evolution of a physical system.
+These are closely connected concepts - after all, many computational models are just solving a given mathematical model numerically. However, the distinction is important. Some computational models can be simulating many different aspects of a physical system, which could involve solving many different mathematical models.
```
Comment (LourensVeen) [inline episodes/01-introduction.md]: Maybe mention than "verification" is about comparing the two and making sure the implementation is correct?

File: episodes/01-introduction.md [hunk 2]
```diff
@@ -1,101 +1,94 @@
...
+### Example solution
 
-
-## Challenge 2: how do you nest solutions within challenge blocks?
-
-:::::::::::::::::::::::: solution 
-
-You can add a line with at least three colons and a `solution` tag.
+- Local climate-ecology: One model simulates the expected climate of a region of the Earth under certain vegetation/land use conditions. A separate model simulates the proliferation and behaviour of animals and vegetation in a given region under influence of various climatic conditions.
+- Stent in a blood vessel: A computational fluid dynamics model (simulating the blood flow) through a metal alloy stent. The stresses created by the blood on the stent are passed to a mechanical model of the stent, and used to deform the stent accordingly. The current deformation state of the stent is passed to the blood flow model.
```
Comment (LourensVeen) [inline episodes/01-introduction.md]: Do stents get deformed by blood flow?

File: episodes/01-introduction.md [hunk 3]
```diff
@@ -1,101 +1,94 @@
...
 
 :::::::::::::::::::::::::::::::::
-::::::::::::::::::::::::::::::::::::::::::::::::
 
-## Figures
 
-You can use standard markdown for static figures with the following syntax:
+## Why couple models together?
```
Comment (LourensVeen) [inline episodes/01-introduction.md:64]: Also: reuse existing software, flexibility due to modularity; often in science you want to compare different models and it's nice if you don't have to maintain two copies of a code base

File: episodes/01-introduction.md (hunk 1 above)
Comment (raar1) [reply episodes/01-introduction.md]: Good point. I've added this to the end of the paragraph.

File: episodes/01-introduction.md (hunk 2 above)
Comment (raar1) [reply episodes/01-introduction.md]: Not really, this was a bit of a lazy example. I have changed it now to be more about in-stent restenosis (still very simplified)

**Comment by LourensVeen** [comment]
Looks good to me!

File: episodes/01-introduction.md [hunk 4]
```diff
@@ -1,101 +1,94 @@
...
+title: "Introduction to model coupling"
+teaching: 30
+exercises: 20
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- What is meant by a "model"?
+- What is the difference between a mathematical model and a submodel?
```
Comment (JaroCamphuijsen) [inline episodes/01-introduction.md]: Is this the difference you want to answer? After reading the rest of the episode I know the difference between mathematical model and computer model, but the relation to the concept of submodel is not so clear. That seems to be linked more closely to the concept of "coupled model", am I right?

File: episodes/01-introduction.md [hunk 5]
```diff
@@ -1,101 +1,94 @@
...
 ::::::::::::::::::::::::::::::::::::: objectives
 
-- Explain how to use markdown with The Carpentries Workbench
-- Demonstrate how to include pieces of code, figures, and nested challenge blocks
+- Explain the benefits of coupling simulation models
 
 ::::::::::::::::::::::::::::::::::::::::::::::::
 
 ## Introduction
+Computer simulations are widely used in modern science to understand and predict the behaviour of natural systems. The computer program that simulates the system is referred to as a "model" of that system. Some models may be very simple e.g. describing the evolution of a 1 dimensional system. Others may be highly complex, involving many different physical processes. A well known use of computer simulation is in the field of weather forecasting, in which a very complex model composed of a multitude of different physical processes must be simulated by a computer program, generally requiring powerful supercomputers. The computer program that simulates a given physical system is known as a _model_ of that system. A given climate simulation code is a _model_ of the Earth's climate, for example.
```
Comment (JaroCamphuijsen) [inline episodes/01-introduction.md]: "The computer program that simulates the system is referred to as a "model" of that system."
"The computer program that simulates a given physical system is known as a _model_ of that system."
These two sentences are almost identical, maybe skip one?

**Review by JaroCamphuijsen — APPROVED** [review]
After the two minor remarks, feel free to merge!

File: episodes/01-introduction.md (hunk 4 above)
Comment (raar1) [reply episodes/01-introduction.md]: Ah good catch, sorry. No, this was a mistake in the question - I'll correct that now.

File: episodes/01-introduction.md (hunk 5 above)
Comment (raar1) [reply episodes/01-introduction.md]: Yes one of these should go. I'll merge both into the first occurence

**Pull request merged**